instrumentation/opensearch/opensearch-java-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opensearch/java/v3_0/OpenSearchJavaAttributesGetter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opensearch.java.v3_0;
+
+import io.opentelemetry.instrumentation.api.incubator.semconv.db.DbClientAttributesGetter;
+import io.opentelemetry.semconv.incubating.DbIncubatingAttributes;
+import javax.annotation.Nullable;
+
+final class OpenSearchJavaAttributesGetter
+    implements DbClientAttributesGetter<OpenSearchJavaRequest, Void> {
+
+  @SuppressWarnings("deprecation") // using deprecated DbSystemIncubatingValues
+  @Override
+  public String getDbSystem(OpenSearchJavaRequest request) {
+    return DbIncubatingAttributes.DbSystemIncubatingValues.OPENSEARCH;
+  }
+
+  @Deprecated
+  @Override
+  @Nullable
+  public String getUser(OpenSearchJavaRequest request) {
+    return null;
+  }
+
+  @Override
+  @Nullable
+  public String getDbNamespace(OpenSearchJavaRequest request) {
+    return null;
+  }
+
+  @Deprecated
+  @Override
+  @Nullable
+  public String getConnectionString(OpenSearchJavaRequest request) {
+    return null;
+  }
```

Comment:
you can remove this, there is a default implementation in `DbClientCommonAttributesGetter`